### PR TITLE
agentless-config

### PR DESCRIPTION
**Pull request by jessegoodier**
Updated both aws amp guides for consistency.

### CODE REVIEW

File: aws-amp/agentless/README.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+
+1. Update all configuration files with your cluster name (replace all `YOUR_CLUSTER_NAME_HERE`). The examples use the key of `cluster` for the cluster name. You can use any key you want, but you will need to update the configmap and deployment files to match. A simplified version of the ADOT DS installation is below.
+
+1. Build the configuration variables:
+
+  ```sh
+  CLUSTER_NAME=YOUR_CLUSTER_NAME_HERE
+  CLUSTER_REGION=us-east-2
+  WORKSPACE_ID=ws-a8c57716-3efb-4fab-ab90-13e6be2a0280
+  WORKSPACE_ARN=$(aws amp describe-workspace --workspace-id ws-a8c57716-3efb-4fab-ab90-13e6be2a0280 --output json | jq -r .workspace.arn)
```
Comment (srpomeroy) [inline aws-amp/agentless/README.md:29]: We should replace the workspace ID with a fake value.

File: aws-amp/agentless/README.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+## Agentless AMP Configuration
+
+1. Update all configuration files with your cluster name (replace all `YOUR_CLUSTER_NAME_HERE`). The examples use the key of `cluster` for the cluster name. You can use any key you want, but you will need to update the configmap and deployment files to match. A simplified version of the ADOT DS installation is below.
+
+1. Build the configuration variables:
+
+  ```sh
+  CLUSTER_NAME=YOUR_CLUSTER_NAME_HERE
+  CLUSTER_REGION=us-east-2
+  WORKSPACE_ID=ws-a8c57716-3efb-4fab-ab90-13e6be2a0280
```
Comment (srpomeroy) [inline aws-amp/agentless/README.md:28]: Not used. Should not contain an actual workspace id.

**Review by srpomeroy — CHANGES_REQUESTED** [review]
Looks good Jesse! I definitely want to have a conversation with you around ADOT/agentless.

**Comment by jessegoodier** [comment]
good finds!
Going to reorganize this whole thing, but already updated with what you found.

**Pull request merged**